root/components/home/Explore.js

```diff
@@ -0,0 +1,172 @@
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/About/History"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('History')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Robert Kaye founded MusicBrainz as a
+                  free open music database. The project has since grown
+                  rapidly from a one-man operation to an international
+                  community of enthusiasts that appreciate
+                  both music and music metadata.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/MusicBrainz_Database"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('The Database')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  The MusicBrainz Database stores all of the
+                  various pieces of information we collect
+                  about music, from artists and their
+                  releases to works and their composers,
+                  and much more.
+                  All our data is available for commercial licensing.
+                  If you are interested in licensing this data
+                  for commercial use, please contact us.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/Editing_FAQ"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Breadth')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  MusicBrainz exists to collect as much
+                  information about music as we can. We do not
+                  discriminate or prefer one “type” of
+                  music over another. Whether it is published or
+                  unpublished, popular or fringe, western or non-western,
+                  human or non-human — we want it all in MusicBrainz.
```

Comment:
@mwiencek, should "Western" be capitalized here? It kinda feels like it but I'd like native speaker confirmation :) 

Why is type in quotes? Just to indicate how wide the concept of type is meant to be? I don't mind it, just making sure since it surprised me a bit at first!

Reply:
Done!